packages/react-components/src/stories/Promo.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { ComponentMeta } from '@storybook/react';
+
+import {
+  IPromoProps,
+  Promo as PromoComponent,
+  PromoSize,
+} from '../components/Promo';
+
+export default {
+  title: 'Components/Promo',
+  component: PromoComponent,
+} as ComponentMeta<typeof PromoComponent>;
+
+type IPromoArgs = IPromoProps;
+
+export const Promo = (args: IPromoArgs): React.ReactElement => (
```

Comment:
Instead of exporting one story I would add more of them to show different variants. It will be helpful for future visual testing.

We can go with one "default" story (for the playground). And variants with:

- small with link
- medium with button
- large with link
- large with button

grouping story with all variants (to display them in docs)
What do you think?

Reply:
Promo stories are updated